ROADMAP MAR18-SEP18.md

```diff
@@ -13,46 +13,60 @@ Here are my current thoughts on a roadmap for our overall engineering organziati
 ## Overall
 
 * Open Source Basic Training - this will be a guide to getting started with Open Source.  This needs to include lessons (links to particularly good blog post tutorials, etc. are welcome) on Git and Github.  Git is often a major hurdle to getting started - we need strong and very visible learning materials on the essentials of git.  We also need a repo that users can immediately contribute to to put their git and Github skills in action - perhaps a repo of text files that a user can add their name or something else to?  The goal of this basic training is to familiarize someone with the workflow of git and Github - which is critical to contributing to any or our projects.  This should also include a brief guide to each of the major Operation Code Open Soure Projects - what technology they use, links to 101 guides to those technologies, etc.  Members should receive a link to this training immediately when they join the org.
+ * _still a big priority -conrad_
+ * _we have the pieces in place but need to get them all together -ashley_
```

Comment:
@nellshamrell - Need a central place for the open source contribution stuff to live
@jjhampton - Need to get people to CONTRIBUTING.md
@kylemh - maybe a webpage for contributing through the main site?
@nellshamrell - a central place for where people can get started for contributions, where I learn about Git and Github. START_HERE has some of it, but it needs to get more traffic
@jjhampton - webpage would be awesome for this

Reply:
How do we feel about pushing some of this off into existing public resources? 

e.g. evaluating and recommending solutions like the top 3 google hits: https://opensource.guide/how-to-contribute/ or https://www.firsttimersonly.com/ or https://medium.freecodecamp.org/how-to-make-your-first-open-source-contribution-in-just-5-minutes-aaad1fc59c9a

Reply:
I agree. We shouldn't try to re-invent the wheel in educating people on `git`. Our website and documentation should be specific to our repository, and link to other resources for general topics like how to "React" and "Git" and "GitHub"